debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java

```diff
@@ -21,6 +21,7 @@
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.regex.Pattern;
 
+import io.debezium.connector.postgresql.*;
```

Comment:
This should be moved near the imports connected to io.debezium.connector.postgresql
Instead of importing .* , could the specific classname be specified